Publish GDocs document to Drupal

anantjain6 opened the pull request:
I have integrated these codes in Google Docs https://docs.google.com/document/d/1fvvySs-eXZf3KxBonJDNUcRBP8aro6ZKTZUrda5DQz4/edit?usp=sharing
1. Make a copy of this document.
3. From the main menu select Add-On > Publish to Drupal > Manage Sites
4. From "Google Docs" Drupal Plugin generate and save an API Key, and enter that API Key & Drupal Site URL in Google Docs Add-On sidebar.
5. From the main menu, Add-On > Publish to Drupal > Publish
6. Select the site from the drop-down and select the "Continue" button
7. Update summary (optional), select content type (article or basic-page), user/admin and to publish or just save this content on Drupal.
8. Submit the form

anantjain6 committed:
Publish GDocs document to Drupal

dbjpanda commented:
![Screen Shot 2019-07-13 at 12 35 03 AM](https://user-images.githubusercontent.com/17483092/61152466-21627c00-a506-11e9-99f4-b297e3b82a70.png)
![Screen Shot 2019-07-13 at 12 34 04 AM](https://user-images.githubusercontent.com/17483092/61152467-21627c00-a506-11e9-841b-6c7faa76f6ed.png)

1. I tried posting two articles.  As you can see the screenshot, it seems the content field not receiving the  real text from gdoc. Field mapping probably not working. Summary fields works but text field always become 404. 
2. What is the procedure to host it in Google Marketplace ? How much it costs ? 
3. Improve UI E.g Check box for site selection etc

anantjain6 committed:
Sugestion & Fix

anantjain6 commented:
The content was not getting published due to lack of permission to Add-On. I fixed it in commit 
4e64dca and done some changes to improve the < select > tag.
I'm following design according to https://developers.google.com/gsuite/add-ons/guides/css

Also, I have updated the Add-On code in doc https://docs.google.com/document/d/1fvvySs-eXZf3KxBonJDNUcRBP8aro6ZKTZUrda5DQz4/edit?usp=sharing

Previously Google Doc Add-On gets published like a chrome extension
But recently Google has introduced Google Suite Marketplace for Docs, Sheet etc Add-On
I'm looking into it and shortly let you know the cost & procedure

anantjain6 commented on Code.gs:
```diff
@@ -99,6 +98,7 @@ function deleteSite(url) {
 function publishContent(summary, type, user, status, url) {
   var doc = DocumentApp.getActiveDocument();
   var title = doc.getName();
+  var folder = DriveApp.getRootFolder();
```
@ankitjain28may In reference to your comment https://github.com/dbjpanda/collaborative_editors/pull/12#issuecomment-511682677
I missed adding this line in that Google Docs document
Sorry for such a silly mistake
Please try now
